packages/babel-traverse/src/scope/index.js

```diff
@@ -864,6 +868,14 @@ export default class Scope {
   }) {
     let path = this.path;
 
+    if (path.isPattern()) {
+      // We are inside some function parameters
+      throw path.buildCodeFrameError(
+        "[Internal error] It's not possible to inject a variable in function parameters. " +
+          "Please check in the stack trace of this error which plugin is causing it.",
```

Comment:
```suggestion
          "Please check this error's stack track to see which plugin is causing it.",
```

Sidebar: it'd be much nicer to tell them which plugin it was

Reply:
Should we attempt parsing the error stack trace using a regular expression?